charts/launch-agent/values.yaml

```diff
@@ -63,6 +63,11 @@ additionalSecretEnvVars:
 # repos. Example: https://username:password@example.com
 gitCreds: |
 
+# Secret name for git SSH credentials.
+# kubectl create secret generic <gitSSHCredsSecret> --from-file=<sshKeySecretKey>=/path/to/keyfile
+gitSSHCredsSecret: ""
+sshKeySecretKey: ""
```

Comment:
I'm also wondering if we should be using a secret of type `kubernetes.io/ssh-auth` instead of an opaque secret. This secret format also requires that the private key be be stored under `ssh-privatekey` https://kubernetes.io/docs/concepts/configuration/secret/#ssh-authentication-secrets